dali/base/dafdesc.hpp

```diff
@@ -327,6 +327,13 @@ interface IStoragePlaneAlias: extends IInterface
     virtual const char *queryPrefix() const = 0 ;
 };
 
+enum StorageType
```

Comment:
Design: Not 100% sure this is a good idea (rather than returning the api name) because it does not lend itself to  supporting pluggable new APIs.  However that is unlikely to be a short term problem, so ok as it stands.

Reply:
It's returning the storage type not the api name.  It needs to know the storage type to work out which API to use... will look into a better way of doing this later.